src/main/drivers/flash_m25p16.c

```diff
@@ -243,20 +251,32 @@ bool m25p16_init(const flashConfig_t *flashConfig)
     return m25p16_readIdentification();
 }
 
+void m25p16_setAddress(bool useLongAddress, uint32_t address, uint8_t *buf)
+{
```

Comment:
Common C ordering is dest, source, size.
m25p16_setAddress(uint8_t *buf, uint32_t address, bool useLongAddress)